internal/ports/convert.go

```diff
@@ -54,25 +55,11 @@ func (c *Client) Convert() *cobra.Command {
 		RunE: func(cmd *cobra.Command, args []string) error {
 			cmd.SilenceUsage = true
 
-			if f := cmd.Flag("bonded"); f.Changed {
-				_, _, err := map[bool]func(string, bool) (*packngo.Port, *packngo.Response, error){
-					true:  c.PortService.Bond,
-					false: c.PortService.Disbond,
-				}[bonded](portID, bulk)
-				if err != nil {
-					return fmt.Errorf("failed to change port bonding: %w", err)
-				}
-			}
-			addrs := []packngo.AddressRequest{{AddressFamily: 4, Public: false}}
-
-			if f := cmd.Flag("public-ipv4"); f.Changed {
-				addrs = append(addrs, packngo.AddressRequest{AddressFamily: 4, Public: true})
-			}
-			if f := cmd.Flag("public-ipv6"); f.Changed {
-				addrs = append(addrs, packngo.AddressRequest{AddressFamily: 6, Public: true})
+			if err := portBondingHandler(cmd, c, portID); err != nil {
```

Comment:
This change in logic is preventing `-2` from converting a port to Layer2. It will try to disbond instead.

Reply:
yes, that right. Missed this case. Fixed it